docker/nginx.conf

```diff
@@ -19,102 +21,115 @@ http {
       set $pass http://ui;
       proxy_pass $pass;
       proxy_hide_header Content-Security-Policy;
-      proxy_set_header Host ingress;
+      proxy_set_header Host taskcluster;
     }
     location /login {
       set $pass http://web-server-web:3050;
       proxy_pass $pass;
       proxy_hide_header Content-Security-Policy;
-      proxy_set_header Host ingress;
+      proxy_set_header Host taskcluster;
     }
     location /graphql {
       set $pass http://web-server-web:3050;
       proxy_pass $pass;
       proxy_hide_header Content-Security-Policy;
-      proxy_set_header Host ingress;
+      proxy_set_header Host taskcluster;
     }
     location /subscription {
       set $pass http://web-server-web:3050;
       proxy_pass $pass;
       proxy_hide_header Content-Security-Policy;
-      proxy_set_header Host ingress;
+      proxy_set_header Host taskcluster;
+    }
+    location /public-bucket/ {
```

Comment:
this was needed to make UI fetch artifact without CORS issues and redirect problems